### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mzouros**

### CODE REVIEW

File: eic-registry-rest/src/main/java/eu/einfracentral/controllers/registry/ProviderController.java [hunk 1]
```diff
@@ -289,7 +291,7 @@ public ResponseEntity<Paging<?>> getRejectedResources(@PathVariable("id") String
                                                           @RequestParam String resourceType, @ApiIgnore Authentication auth) {
         allRequestParams.add("resource_organisation", providerId);
         allRequestParams.add("status", "rejected resource");
-        allRequestParams.add("published", "false");
+        allRequestParams.add("published", false);
```
Comment (spyroukostas) [inline eic-registry-rest/src/main/java/eu/einfracentral/controllers/registry/ProviderController.java:294]: Does this work as intended?

Comment (mzouros) [reply eic-registry-rest/src/main/java/eu/einfracentral/controllers/registry/ProviderController.java:294]: Seems like it works both ways

File: eic-registry-rest/src/main/java/eu/einfracentral/controllers/registry/ServiceController.java [hunk 2]
```diff
@@ -304,7 +304,7 @@ public ResponseEntity<ServiceBundle> auditService(@PathVariable("id") String id,
     @PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_EPOT')")
     public ResponseEntity<Paging<?>> getRandomResources(@ApiIgnore @RequestParam Map<String, Object> allRequestParams,
                                                         @ApiIgnore Authentication auth) {
-        FacetFilter ff = FacetFilterUtils.createFacetFilter(allRequestParams);
+        FacetFilter ff = new FacetFilter();
```
Comment (spyroukostas) [inline eic-registry-rest/src/main/java/eu/einfracentral/controllers/registry/ServiceController.java:307]: Replace all occurrences of:

```
FacetFilter ff = new FacetFilter();
ff.setQuantity(allRequestParams.get("quantity") != null ? Integer.parseInt((String) allRequestParams.remove("quantity")) : 10);
ff.setFilter(allRequestParams);
```

with:

`FacetFilterUtils.createFacetFilter(allRequestParams);`

File: eic-registry/src/main/java/eu/einfracentral/registry/manager/ProviderRequestManager.java [hunk 3]
```diff
@@ -117,7 +118,7 @@ public void sendMailsToProviders(List<String> serviceIds, EmailMessage message,
             providerContactNames.remove(key);
 
             String providerSubject = String.format("[%s] You have a new message from user [%s]-[%s], considering the Provider [%s]", projectName, message.getSenderName(), message.getSenderEmail(), entry.getKey());
-            String userSubject = String.format("[%s] Your message considering the CatRIS Services has been sent successfully", projectName);
+            String userSubject = String.format("[%s] Your message considering the EOSC Services has been sent successfully", projectName);
```
[the remark this answers is not in the document]
Comment (spyroukostas) [reply eic-registry/src/main/java/eu/einfracentral/registry/manager/ProviderRequestManager.java:121]: Search and replace hard-coded "EOSC" values.

**Review by spyroukostas — CHANGES_REQUESTED** [review]
See attached comments

**Review by spyroukostas — APPROVED** [review]

**Pull request merged**